cypress/integration/header.spec.ts

```diff
@@ -1,57 +1,40 @@
 /// <reference path="../support/index.d.ts" />
 
+const testNavButton = (buttonId: string, url: string) => {
+  cy.get(`[data-cy=nav-${buttonId}]`).click()
+  cy.url().should('eq', Cypress.config().baseUrl + `${url}`)
+}
+
+const createNavTest = (msg: string, buttonId: string, url: string) =>
```

Comment:
I think you normally want tests to be immediately readable vs less code duplication. 
I think we should just have the it() inside the testNavButton for simplicity.

Reply:
Strongly agree here, It's not like it actually adds anything.
done a34191d